extensions/container-image/container-image-docker/deployment/src/main/java/io/quarkus/container/image/docker/deployment/DockerFileBaseInformationProvider.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.container.image.docker.deployment;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+
+interface DockerFileBaseInformationProvider {
```

Comment:
I am wondering if this should be moved to `spi` or a `commons` module as we could possibly use it in other places too, e.g. `container-image-openshift` when we use the docker build strategy.

Reply:
I would say let's keep it here for now and if we do need it, then we can move. My point is that making it more general later is easier than making it part of an SPI now and potentially having to break down the line if this doesn't fit the bill for other use cases.